JetUtilities/src/crystalBall.cc

```diff
@@ -0,0 +1,227 @@
+#include "JetMETAnalysis/JetUtilities/interface/crystalBall.h"
+
+#include "TMath.h"
+
+double sum_gaus_fnc(double *x, double *par); // sum of 3gaus+pol7
+
+double sum_gaus_pol_fnc(double *x, double *par); // sum of 3gaus+pol7
+
+double exp_fcn(double x, double norm, double slope); // exp 
+
+double gauss_fcn(double x, double norm, double mean, double sigma); // gauss 
+
+double pol_fcn(double x, double p0, double p1=0, double p2=0, double p3=0, double p4=0, double p5=0, double p6=0, double p7=0); // pol7
+
```

Comment:
Why are these functions declared here and not in the header file? Also, can we make the comment style similar to the existing code?

For example:
/// commend about the function to be declared below
double function_name();

/// Other comment
double other_function_name()